cmd/juju/application/bundle/bundle.go

```diff
@@ -232,27 +232,41 @@ func applicationConfigValue(key string, valueMap interface{}) (interface{}, erro
 
 // ComposeAndVerifyBundle merges base and overlays then verifies the
 // combined bundle data.
```

Comment:
Since it is uncommon to have a `[]error` return we probably should have a clear documentation message here saying "this returns a list of warnings encountered while processing the bundle"